app/src/main/java/de/grobox/transportr/departures/DepartureViewHolder.java

```diff
@@ -126,9 +126,9 @@ private void bindDelay(long delayTime) {
 			delay.setVisibility(GONE);
 		} else {
 			delay.setText(getDelayString(delayTime));
-			delay.setVisibility(VISIBLE);
-			if (delayTime <= 0) delay.setTextColor(ContextCompat.getColor(delay.getContext(), R.color.md_green_500));
-			else delay.setTextColor(ContextCompat.getColor(delay.getContext(), R.color.md_red_500));
+            delay.setVisibility(VISIBLE);
+            if (delayTime <= 0) delay.setTextColor(ContextCompat.getColor(delay.getContext(), R.color.md_green_500));
+            else delay.setTextColor(ContextCompat.getColor(delay.getContext(), R.color.md_red_500));
```

Comment:
I'm not the dev but I'm sure this tabs to spaces was not intended and should be reverted.

Reply:
you are right. I will fix this right now